api4/system.go

```diff
@@ -17,6 +18,8 @@ func InitSystem() {
 	l4g.Debug(utils.T("api.system.init.debug"))
 
 	BaseRoutes.System.Handle("/ping", ApiHandler(getSystemPing)).Methods("GET")
+	BaseRoutes.System.Handle("/health", ApiSessionRequired(healthCheck)).Methods("GET")
```

Comment:
Why the extra route, I think we should just override the ping, which is what this is meant for.